cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialID.java

```diff
@@ -86,12 +86,15 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 				id = seqValue + 1;
 			}
 			else {
+				if (startValue.charAt(0) == '0') {
```

Comment:
If the condition you want to check is related to numbers I would always prefer handling it as a number.
What about negative integers? I guess they still should not be accepted?
I would still see no reason though, why zero could not be a valid start value.

When you convert the string to an integer something like leading zeros does not matter anymore anyhow, because you work with a number representation and not a string anymore.